frontend/src/routes/Governance/common/util.tsx

```diff
@@ -732,19 +732,22 @@ export function getEngineWithSvg(apiGroup: string): JSX.Element {
   }
 
   return (
-    <div style={{ display: 'flex', alignItems: 'center', gap: 8 }}>
+    <>
```

Comment:
I didn't know about this--looks like unnamed HTML tags are a thing now? What's the benefit? Is it an alias for `div` or something?

Reply:
@dhaiducek in React, it's an alias for `React.Fragment` which is a JSX.Element but does not impact the DOM as far as I know. It's just a way to return multiple HTLM elements as a single JSX.Element.